packages/api/test/e2e/promise-metadata.spec.js

```diff
@@ -0,0 +1,68 @@
+    const genesisHash = '0x8579ded4438b26597e188cda8579fdd6abf09b74ab672e604b9584cd8b4c3d0b';
+    const specVersion = '62';
+    const prebundles = {};
+    const key = `${genesisHash}-${specVersion}`;
+    prebundles[key] = rpcData;
+    const api = await ApiPromise.create({
+      prebundles
+    });
+
+    expect(api.genesisHash).toBeDefined();
+    expect(api.runtimeMetadata).toBeDefined();
```

Comment:
`expect api.runtimeMetadata.toHex()` to be rpcData

Reply:
![Screen Shot 2019-04-30 at 10 58 24 AM](https://user-images.githubusercontent.com/29415595/56932380-a3621c00-6b37-11e9-91c1-ec7d2221e303.png)
api.runtimeMetadata.toHex() is not supported, throws throw as shown in the screenshot attached.